src/traces/sankey/plot.js

```diff
@@ -125,13 +125,14 @@ module.exports = function plot(gd, calcData) {
 
     var linkHoverFollow = function(element, d) {
 
+        var rootBBox = gd.getBoundingClientRect();
```

Comment:
Can you use `fullLayout.width` and `fullLayout.height` or even `fullLayout._size` instead? Those `getBoundingClientRect` calls are expensive.